python/py-spopt/Portfile

```diff
@@ -60,7 +58,20 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-spaghetti \
                     port:py${python.version}-tqdm
 
+    if {${python.version} == 39} {
+        PortGroup       github 1.0
+
+        github.setup    pysal spopt 0.5.0 v
+
+        python.pep517   no
```

Comment:
why is this set to `no`; subports for Python 3.7+ should ideally build with it set to `yes`? What is the error you see when you leave this to the default?

Reply:
@nilason ?